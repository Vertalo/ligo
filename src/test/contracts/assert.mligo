let main (p: bool) (s: unit) =
  let u : unit = assert p
  in ([] : operation list), s
