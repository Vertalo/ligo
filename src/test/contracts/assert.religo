let main = (p: bool, s: unit) => {
  let u: unit = assert(p);
  ([]: list(operation), s);
};
