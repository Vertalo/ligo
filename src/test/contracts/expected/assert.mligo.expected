let main (p, s : bool * unit) =
  let u : unit = assert p in
  ([] : operation list), s
